---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

[Skills](#sk) &nbsp; &nbsp; - &nbsp; &nbsp; [Education](#ed) &nbsp; &nbsp; - &nbsp; &nbsp; [Work experience](#wrkexp) &nbsp; &nbsp; - &nbsp; &nbsp; [Online Courses](#onlinecources) &nbsp; &nbsp; - &nbsp; &nbsp; [Awards](#aw) &nbsp; &nbsp;


<h2 id="sk">
</h2>

Skills
======
 ·Data Science ·Machine Learning ·NGS data analysis · Statistical Learning · Computational-Genomics · R · Python · High Performance Computing (HPC) · LATEX 
 
<h2 id="ed">
</h2>

Education
======
* **PhD** (Computational Science), Jawaharlal Nehru University, New Delhi, India, 2010-2017
  * ***Work:** Applied various Statistical methods to study genomics features and proposed an improved Machine learning approach that effectively utilises high-dimensional data to find patterns in the genome.*
* **M. Tech.** (Computational and Systems Biology), Jawaharlal Nehru University,  New Delhi, India, 2008-2010
  * ***Work:** Application of Signal processing techniques to detect cell boundaries in the digital microscopy images.*
* **B. Tech.** (Biotechnology), Kurukshetra University, Kurukshetra, India, 2003-2007
  * ***Work:**  Besides my coursework, In my dissertation, I modelled cell growth in a closed and controlled fermentation environment.
 
<h2 id="wrkexp">
</h2>

<!---  <details><summary>Expand</summary><blockquote> # will work if we make "parse_block_html: false" in yaml file  -->

Work experience
======
* **2023/05- Present: Postdoctoral Research Scientist, Aarhus University, Denmark**
  <details><blockquote>

  * **Responsibilities:** Systematically strategizing and executing research projects, leading to the unveiling of groundbreaking discoveries shared with the scientific community via publication or comprehensive reporting.
  * **Project 1:**  Development of computational methods to discover genomic patterns responsible for rare diseases in newborns.
  * **Skills:** Data Science · NGS data analysis · Genomics · R · High Performance Computing (HPC)

  </blockquote></details>
  
* **2020/03- 2023/04: Researcher, University of Gothenburg, Sweden**
  <details><blockquote>

  * **Responsibilities:** Conducting thorough planning and executing research projects, culminating in the dissemination of novel discoveries to the scientific community through publication or reporting.
  * **Project 1:** My research delves into the rich diversity of mutation and epigenomic data characterizing cancer cells and their roles in cancer progression, particularly focusing on melanoma. I aim to advance our understanding by employing sophisticated data normalization techniques to seamlessly integrate multimodal datasets. Through this integration, I strive to develop an innovative artificial intelligence (AI)-driven clinical decision-making system tailored for precise treatment guidance in melanoma patients. By harnessing the power of AI, we aim to revolutionize cancer care by offering personalized and effective therapeutic strategies. 
  * **Project 2:** This is a collaborative project where I used barseq (barcode high throughput sequencing) data of yeast deletion strains to discover the genes responsible for delay in cell division in mitochondria-depleted cells.
  * **Skills:** Data Science · NGS data analysis · Genomics · R · High Performance Computing (HPC)

  </blockquote></details>

* **2019/11-2020/03: Post-Doctoral researcher, Albert Einstein College of Medicine, New York, USA**
  <details><blockquote>

  * **Responsibilities:** Planning and completion of research projects, followed by publishing or reporting new findings to the scientific community.
  * **Project 1:** New DNA sequencing technologies, such as single-cell RNA-seq, produce very high-dimensional (~30,000 x 1,000,000) datasets, visualizing this data is challenging but useful to understand many biological processes. So, I have developed an interactive tool named “Stereo3D” that reduces the dimension of data to 3D and produces interactive stereo images for better visualization. ([Publication](https://academic.oup.com/bioinformatics/article/36/14/4189/5838181), [Stereo3D](https://github.com/bioinfoDZ/Stereo3D))
  * **Project 2:**  Developed a software tool named "HSMotifDiscover" and Its app, to discover patterns in specific type of biological sequences. This tool uses a specialized version of the Gibbs-sampling optimization algorithm to detect the conserved patterns in the sequences. A detailed description of the work can be found in the publication section below.([Publication](https://academic.oup.com/bioinformatics/article/38/16/4036/6623407?searchresult=1), [HSMotifDiscover](https://github.com/bioinfoDZ/HSMotifDiscover), [HSMotifDiscoverApp](https://hsmotifdiscover.shinyapps.io/HSMotifDiscover_ShinyApp/))
  * **Skills:** Statistical Learning · Software dovelopemnt · Gibbs-Sampling optimisation · Rotational Matrices · R

  </blockquote></details>

* **2018/03-2019/11: Post-Doctoral researcher, Rutgers cancer Institute of New Jersey, New Jersey, USA**
  <details><blockquote>

  * **Responsibilities:** Planning and completion of research projects, followed by publishing or reporting new findings to the scientific community.
  * **Research project:** Used Statistical learning techniques such as  Non-Negative matrix factorization technique and Hidden Markov model (HMM), to find unique mutational patterns associated with the defective replication process in the cancer genome. This information is useful in detecting cancers related to defective replication processes for precise treatment of cancer patients. An R-package named MutSigTools was also developed to find mutational patterns related to other mutational sources. ([Publication](https://www.nature.com/articles/s42003-020-01119-5), [MutSigTools](https://github.com/sjdlabgroup/MutSigTools))
  * **Skills:** Dimensionality reduction · Statistical Learning · Non-Negative Matrix Factorisation (NMF) · Hidden Markov Models(HMM) · Python · Genomics · R · High Performance Computing (HPC)

  </blockquote></details>

* **2017/02-2017/06: Senior Research Scientist (Machine Learning), Valiance Solutions Pvt. Ltd, Noida, India**
  <details><blockquote>

  * **Responsibilities:** Guided a team of data scientists in implementing Machine learning algorithms and troubleshooted the dataset-specific problems.
  * **Project 1:** Contributed to the development of a logistic regression based Credit risk model for credit risk agency
  * **Project 2:** Contributed to the development of an Intelligent system for customer sentiment analysis based on reviews using deep learning techniques. (Detailed Project description is given at the [link](https://valiancesolutions.com/whitepapers/sentiment-analysis/) )
  * **Project 3:** Feasibility study of deep learning-based optical character recognition technique to read hand-filled forms.
  * **Skills:** Deep Learning · Machine Learning · Python · R

  </blockquote></details>

Publications
======
* [Google Scholar](https://scholar.google.co.in/citations?user=2482OI4AAAAJ&hl=en&authuser=1)

<h2 id="onlinecources">
</h2>

Online Courses
======
 * Machine Learning (2013) : Coursera [Certificate](/files/Coursera_introduction_to_machine_learning.pdf)
 * Linear Integer Programming (2014) : Coursera [Certificate](/files/Coursera_linear_integer_programming_2014.pdf)
 * Computational Molecular Evolution (2014) : Coursera [Certificate](/files/Coursera_computational_molecular_evolution_2014.pdf)
 * Introduction to Systems Biology (2014) : Coursera [Certificate](/files/Coursera_Introuction_to_systems_Biology_2014.pdf)

<h2 id="aw">
</h2>

Awards
======
 * Senior Research Fellowship (2014-2016) : Transdisciplinary Research, Council of Scientific and Industrial Research (CSIR), India
 * Graduate Aptitude Test in Engineering 2011: Biotechnology, Indian Institute of Technology Madras (IIT Madras), India


